rundeckapp/grails-app/jobs/rundeck/quartzjobs/ExecutionsCleanUp.groovy

```diff
@@ -188,33 +210,60 @@ class ExecutionsCleanUp implements InterruptableJob {
 
     private int totalAllExecutions(ExecutionService executionService, String project){
         ExecutionQuery query = new ExecutionQuery(projFilter: project)
-        Map result = executionService.queryExecutions(query)
-        return null != result ? result.total : 0
+        def criteriaClos = { isCount ->
```

Comment:
this param `isCount` doesn't seem to be used?

Reply:
your right, I removed it